packages/components/filters/src/filter-menu/header/header.tsx

```diff
@@ -1,5 +1,73 @@
-function Header() {
-  return <div>header</div>;
-}
+import { useState, ReactNode } from 'react';
+import IconButton from '@commercetools-uikit/icon-button';
+import Spacings from '@commercetools-uikit/spacings';
+import { SortingIcon } from '@commercetools-uikit/icons';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+
+type TOption = {
+  value: string;
+  label: string;
+};
+
+type THeaderProps = {
+  headerLabel: ReactNode;
```

Comment:
what about just `label`? If more labels are added later, they can get the qualifier in their name.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2951/commits/87f00d071d260cc8ba05dc38380993e55887aeca

Thank you